src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/TestHelpers.kt

```diff
@@ -163,8 +218,9 @@ fun randomChangePolicy(
     return ChangePolicy(policyID, state, emptyList())
 }
 
-fun randomDefaultNotification(): Map<String, Any>? { // TODO: DefaultNotification data class
-    return null // TODO: random DefaultNotification
+// will only return null since we dont want to send actual notifications during integ tests
+fun randomErrorNotification(): ErrorNotification? {
+    return null
```

Comment:
NP: Since Kotlin recommends [using expressions bodies for single expression functions](https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting) it may be better to do ```= null```. There are cases where we need to do this elsewhere in our code but that can be handled in upcoming code cleanups/refactors.